index.html

```diff
@@ -101,7 +96,252 @@ <h1>I'm Adeyemi Glad to see you!</h1>
                 </li>  
               </ul>            
             </div>
-            <div> 
+          </div>
+        </section>
+        <Section class="Works">
```

Comment:
Kindly remove the yellow background😉 and make use of the required background in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)